[CI] search.aggregation/235_composite_sorted failures

**Build scan**:

https://gradle-enterprise.elastic.co/s/2w65af7psd4ju

**Repro line**:

```
./gradlew ':qa:mixed-cluster:v7.1.0#mixedClusterTest' \      
  -Dtests.class="org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT" \
  -Dtests.method="test {p0=search.aggregation/235_composite_sorted/two sources - first page}" \
  -Dtests.seed=C13375AE83190C97 \
  -Dtests.bwc=true \
  -Dtests.locale=ar-SD \
  -Dtests.timezone=America/St_Barthelemy \
  -Druntime.java=8
```

**Reproduces locally?**:

Yes

**Applicable branches**:

7.x, 7.13

**Failure history**:

started earlier yesterday night on 7.x and 7.13, apprix 11 failures on both branches in the last half day

**Failure excerpt**:

```
java.lang.AssertionError: Failure at [search.aggregation/235_composite_sorted:133]: expected [2xx] status code but api [search] returned [400 Bad Request] [{"error":{"root_cause":[{"type":"x_content_parse_exception","reason":"[1:118] [date_histogram] unknown field [calendar_interval], parser not found","stack_trace":"[[1:118] [date_histogram] unknown field [calendar_interval], parser not found]; nested: XContentParseException[[1:118] [date_histogram] unknown field [calendar_interval], parser not found];\n\tat 
```

--- a/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/235_composite_sorted.yml
+++ b/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/235_composite_sorted.yml
@@ -1,6 +1,5 @@
 ---
 setup:
-
   - do:
       indices.create:
         index: sorted
@@ -75,8 +74,8 @@ setup:
 ---
 one source - first page:
   - skip:
-      version: "all"
-      reason: "Awaits fix: https://github.com/elastic/elasticsearch/issues/72362"
+      version: " - 7.1.99"
+      reason: calendar_interval added in 7.2.0
   - do:
       search:
         index: sorted
@@ -101,8 +100,8 @@ one source - first page:
 ---
 one source - second page:
   - skip:
-      version: "all"
-      reason: "Awaits fix: https://github.com/elastic/elasticsearch/issues/72362"
+      version: " - 7.1.99"
+      reason: calendar_interval added in 7.2.0
   - do:
       search:
         index: sorted
@@ -129,8 +128,8 @@ one source - second page:
 ---
 two sources - first page:
   - skip:
-      version: "all"
-      reason: "Awaits fix: https://github.com/elastic/elasticsearch/issues/72362"
+      version: " - 7.1.99"
+      reason: calendar_interval added in 7.2.0
   - do:
       search:
         index: sorted
@@ -160,8 +159,8 @@ two sources - first page:
 ---
 two sources - second page:
   - skip:
-      version: "all"
-      reason: "Awaits fix: https://github.com/elastic/elasticsearch/issues/72362"
+      version: " - 7.1.99"
+      reason: calendar_interval added in 7.2.0
   - do:
       search:
         index: sorted

